prime-router/src/main/kotlin/common/PhoneUtilities.kt

```diff
@@ -0,0 +1,59 @@
+package gov.cdc.prime.router.common
+
+import com.google.i18n.phonenumbers.PhoneNumberUtil
+
+/**
+ * Enumeration representing the different parts of a phone number.
+ */
+enum class PhonePart {
+    Country,
+    AreaCode,
+    Local,
+    Extension
+}
+
+/**
+ * A collection of methods for dealing with phone parts.
+ */
+object PhoneUtilities {
+
+    /**
+     *  Parses the [cleanedValue] into a PhoneNumber and returns the [part] requested.
+     *  @return either the part of the phone number requested, or null if the phone number was invalid
+     */
+    fun getPhoneNumberPart(cleanedValue: String, part: PhonePart): String? {
+        val phoneNumberUtil: PhoneNumberUtil = PhoneNumberUtil.getInstance()
```

Comment:
Not sure how much of a performance gain it would be, but this getInstance could be its own static variable.